docs/gateway.md

```diff
@@ -69,6 +69,16 @@ The Scarf Gateway is a free hosted service that is provided to maintainers and u
 
 We are expecting to meet a monthly service uptime percentage of 99.9%. Guarantees of our service-level agreement will be made available in the future, when the system moves from beta to general availability.
 
+### Badges
+
+All packages on the Scarf Gateway offer dynamic Scarf-powered README badges automatically. Head to your package page, and the badges will be shown in the details section near the top. Copy the URL, paste it into your project’s README based on whatever doc format you are using and you’re all set.
+
+**What is the difference between the downloads badge and the company badge?**
```

Comment:
We'll actually need an additional newline between the Q and A here, this currently renders the Q and A on the same line (just how GH flavored markdown works).

Reply:
i made that way intentionally so it would look less like a FAQ, and more like description - but if it looks wrong to you i will adjust. Also going to attempt an image embed...

Reply:
OK, updated with proper spacing. It looks better. thanks. still working on img placement